renderdoc/driver/d3d12/d3d12_device.h

```diff
@@ -997,6 +997,7 @@ class WrappedID3D12Device : public IFrameCapturer, public ID3DDevice, public ID3
 
   RDResult ReadLogInitialisation(RDCFile *rdc, bool storeStructuredBuffers);
   void ReplayLog(uint32_t startEventID, uint32_t endEventID, ReplayLogType replayType);
+  void ReplayDraw(ID3D12GraphicsCommandListX *cmd, const ActionDescription &action);
```

Comment:
I wonder if it makes sense to keep this inside `D3D12_PixelHistory` for now as it is only partially implemented.

The pixel history code can line up with the missing implementation.

Then when the implementation is complete, it can be moved to `WrappedID3D12Device`

Reply:
Sure, will move it to d3d12_pixelhistory.cpp